thingsboard_gateway/connectors/bacnet/device.py

```diff
@@ -169,13 +169,40 @@ async def rescan(self):
     @staticmethod
     def find_self_in_config(devices_config, apdu):
         apdu_address = apdu.pduSource.__str__()
+        device_identifier = apdu.iAmDeviceIdentifier[-1]
 
         for device_config in devices_config:
+            if Device.is_device_identifier_match(device_identifier, device_config.get('deviceIdentifier')):
```

Comment:
We are already using `objectId` in configuration, let's name this parameter `deviceId` or `deviceObjectId`.